Dockerfile

```diff
@@ -80,7 +92,8 @@ COPY pnpm-lock.yaml .
 COPY libs/ libs
 COPY frontend/ frontend
 
-RUN npm install -g pnpm@9.4.0
+RUN npm install -g pnpm@9.1.4
+RUN if [ ${PLATFORM} == "arm64" ]; then sed -iz 's/"elm-coverage": "^0.4.1",//g' frontend/package.json; fi
```

Comment:
I'm not using `elm-coverage` so we can fully remove it ^^

Reply:
So is it safe to remove it from the [package.json](https://github.com/azimuttapp/azimutt/blob/4471fde1238ab60af5e9bc625b7e94cfa17b00bc/frontend/package.json#L72) for this PR?

Reply:
Yes, and you will need to run a `pnpm install` from the root folder to update the `pnpm-lock.yaml` accordingly